pkg/controller/sparkapplication/sparkui_test.go

```diff
@@ -153,6 +166,40 @@ func TestCreateSparkUIService(t *testing.T) {
 			},
 			expectError: false,
 		},
+		{
+			name: "service with annotations",
+			app:  app1,
+			annotations: map[string]string{
+				"getambassador.io/config": `|
+---
```

Comment:
Can we define such raw string as a constant at the top to make the indention correct?